justfile

```
@@ -0,0 +1,25 @@
+alias b := build
+alias c := check
+alias f := fmt
+alias t := test
+
+_default:
+   @just --list
+
+# Build the project
+build:
+   cargo build
+
+# Check code: formatting, compilation, linting, and commit signature
+check:
+   cargo +nightly fmt --all -- --check
+   cargo check --all-features --all-targets
```

Does not cover everything, should be:
```suggestion
   cargo check --all-features --all-targets --bins --lib --workspace --tests --benches
```